I2C.md

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
Do we consider having docs under docs/ folder in the repository?  or all in the root?